deploy-edpm.yml

```diff
@@ -79,70 +79,3 @@
   ansible.builtin.import_playbook: playbooks/06-deploy-architecture.yml
   tags:
     - edpm
-
-- name: Run Post-deployment admin setup steps, test, and compliance scan
```

Comment:
@eshulman2 deploy-edpm.yml playbook is used in upstream edpm job. SInce we moved these tasks to post-deployment.yaml, tempest is no longer running in https://softwarefactory-project.io/zuul/t/rdoproject.org/builds?job_name=podified-multinode-edpm-deployment-crc job